src/Components/api.js

```diff
@@ -14,6 +14,15 @@ export const fetchReviews = () => {
       .catch((err) => console.log(err));
   }
 
+  export const fetchComments = (review_id) => {
+    return reviewApi
+      .get(`/api/reviews/${review_id}/comments`)
+      .then((res) => {
+        return res.data;
+      })
+      .catch((err) => console.log(err));
+    }
+    
```

Comment:
Good work with creating a new api call for getting just the comments.

For the catch error I would do `return err` instead of console logging :)